src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -36,66 +25,9 @@ public class ElevatorConstants {
   /** */
   public static final int TOP_MOTOR_CURRENT_LIMIT = 50;
 
-  /** */
-  /** */
-  public static final ElevatorFeedforward FF_MODEL = new ElevatorFeedforward(1, 1, 1, 1);
-
-  /** */
-  public static final PIDController PID_CONTROLLER = new PIDController(1, 0, 0);
-
-  /** */
-  public static final TrapezoidProfile TRAPEZOID_PROFILE =
-      new TrapezoidProfile(new TrapezoidProfile.Constraints(1, 2));
-
-  /** Ramp Rate of the elevator System ID in volts per second */
-  public static final double SYS_ID_RAMP_RATE = 0.2;
-
-  /** Setp Voltage of the elevator System ID in volts */
-  public static final double SYS_ID_STEP_VALUE = 7;
-
-  /** Timeout of the elevator System ID in volts */
-  public static final double SYS_ID_TIMEOUT = 30;
-
   /** How many meters the elevator can be off its goal position for it to be sufficient */
   public static final double ALLOWED_DEVIANCE = 0.01;
 
-  /** */
-  public static final Translation3d TRANSLATION_FROM_ROBOT = new Translation3d(-0.2, 0, 0.255);
```

Comment:
Don't you need this constant for sim? (and eventually optimal inverse kinematics solving)